### ISSUE TITLE
Media3 Exoplayer subtitle font size increase

### ISSUE THREAD

**Issue by simranthakkar1996** [androidx/media#1408]
I need to increase the font size of the subtitle used by media3 exoplayer 

I am not able to find a correct way to increase the default font size of the text
Any suggestion or feedback to increase the subtitle font size would be really helpful.

Because after player view intiliazation I am trying to add the custom font
```
SubtitleView subtitleView = playerView.getSubtitleView();
if (subtitleView != null) {
            CaptionStyleCompat cusomStyle = new CaptionStyleCompat(
                    Color.RED,       // Foreground color
                    Color.TRANSPARENT, // Background color
                    Color.TRANSPARENT, // Window color
                    CaptionStyleCompat.EDGE_TYPE_NONE, // Edge type
                    Color.BLACK,       // Edge color
                    Typeface.DEFAULT_BOLD
            );
            subtitleView.setApplyEmbeddedStyles(false);
            subtitleView.setApplyEmbeddedFontSizes(false);
            subtitleView.setFixedTextSize(TEXT_SIZE_TYPE_ABSOLUTE, 30F);
            subtitleView.setStyle(cusomStyle);
}
```
But its not doing anything 
Thank you for the reply in advance

**Comment by icbaker** [comment]
I took the code snippet you provided and copy-pasted it into the main demo app's `PlayerActivity` just after `playerView` is initialized:
https://github.com/androidx/media/blob/d833d59124d795afc146322fe488b2c0d4b9af6a/demos/main/src/main/java/androidx/media3/demo/main/PlayerActivity.java#L117-L120

I made one correction. You are passing `TEXT_SIZE_TYPE_ABSOLUTE` into a method expecting an int of type [`@Dimension`](https://developer.android.com/reference/androidx/annotation/Dimension), so I changed this to `Dimension.PX`.

I then played the Subtitle > TTML Positioning sample in the demo app and saw large, red subtitles (this has overridden the styling from the source material):

![android-screenshot-2024-05-30T153204](https://github.com/androidx/media/assets/1279752/16b7684e-efbe-4521-b89c-4da5db60542b)

I confirmed that increasing the text size number from `30` to `120` makes the subitles larger:

![android-screenshot-2024-05-30T153732](https://github.com/androidx/media/assets/1279752/5b5a7307-2e3e-41d5-9a00-9f4b826d7e78)

So this seems to be working for me.

I suggest trying it in the demo app too, and seeing if it works for you, and if it does then I suggest spending some time identifying what's different between your app and the demo app.
